marshmallow_dataclass/__init__.py

```diff
@@ -408,6 +408,9 @@ def _internal_class_schema(
         if hasattr(v, "__marshmallow_hook__") or k in MEMBERS_WHITELIST
     }
 
+    # Determine whether we should include non-init fields
+    include_non_init = getattr(getattr(clazz, "Meta", None), "include_non_init", False)
```

Comment:
This would need to be documented...

Reply:
Are you happy with the interface? If so, will go ahead and add some documentation

Reply:
The interface is fine by me. But the tests are failing